packages/reporting-server/README.md

```diff
@@ -88,7 +88,65 @@ postgres://<user>:<password>@<host>/<database>
 for more information, see https://tortoise-orm.readthedocs.io/en/latest/databases.html.
 
 
+## Install PostgreSQL (Optional step if you use a different database)
+
+Instructions for ubuntu 20.04:
+
+Open a terminal and run:
+
+```bash
+sudo apt update && sudo apt install postgresql postgresql-contrib
+```
+
+Once PostgreSQL is installed, you can run the following command to create a database:
+
+``` bash
+sudo -u postgres createdb <database>
+# E.g: sudo -u postgres createdb reporting
+```
+
+For more information you can check [this tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-postgresql-on-ubuntu-20-04-quickstart)
+
+
+## Run the application
```

Comment:
I think this is missing the instructions to set the configs for the db url, user and password.